go/pvl/helpers.go

```diff
@@ -17,12 +17,8 @@ import (
 	jsonw "github.com/keybase/go-jsonw"
 )
 
-// Substitute vars for %{name} in the string.
-// Only substitutes whitelisted variables.
-// It is an error to refer to an unknown variable or undefined numbered group.
-// Match is an optional slice which is a regex match.
-// AllowActiveString makes active_string a valid variable.
-func substitute(template string, state scriptState) (string, libkb.ProofError) {
+// Substitute register values for %{name} in the string.
+func substitute(template string, state scriptState, regexEscape bool) (string, libkb.ProofError) {
```

Comment:
Alternatively, if we think the number of functions would grow too large that way, maybe pass in the escape function as an argument?